docs/calico-with-docker/docker-network-plugin/cloud-config/user-data-first

```diff
@@ -68,16 +56,9 @@ write_files:
     # Can't directly write to .profile since it's a symlink to a RO filesystem
     mkdir -p /opt/bin
     rm /home/core/.bashrc
-    echo 'PATH=/opt/bin:$PATH' > /home/core/.bashrc
+    echo 'PATH=$PATH:/opt/bin' > /home/core/.bashrc
     echo 'export ETCD_AUTHORITY="$private_ipv4:2379"' >> /home/core/.bashrc
     echo 'Defaults env_keep +="ETCD_AUTHORITY"' >>/etc/sudoers.d/etcd
-- path: /home/core/get_latest_docker.sh
```

Comment:
Updated AWS and GCE docs to use correct CoreOS Alpha images.  The DigitalOcean doc explains how to choose the newest alpha image via a UI, so that should be covered.